smooth/components/component_storage_h2.py

```diff
@@ -42,6 +44,12 @@ def __init__(self, params):
 
         # ------------------- UPDATE PARAMETER DEFAULT VALUES -------------------
         self.set_parameters(params)
+        # Initial storage level [kg].
+        self.storage_level_init = self.initial_storage_factor * self.storage_capacity
+        if self.slw_factor is not None:
+            self.storage_level_wanted = self.slw_factor * self.storage_capacity
+        else:
+            self.storage_level_wanted = None
```

Comment:
the storage_level_wanted gets overwritten with none in case it was configured in the model description, that is what you want than delete line 39 to make it not configurable directly, which avoids confusion. Otherwise deleting lines 51 and 52 is an option, but you have to decide wheather slw_factor or direct configuration of storage_level_wanted has priority in case both values are configured